tools/scripts/generate_plantuml_svg.sh

```diff
@@ -21,32 +21,40 @@
 
 set -e
 
+declare -a REQUIRED_PROGRAMS=("plantuml" "git")
+
+STARTUP_CHECK_SUCCESS=true
+verifyProgram() {
+    if ! command -v $1 &>/dev/null
+    then
+        echo Please install \"$1\" to use this script
+        STARTUP_CHECK_SUCCESS=false
+    fi
+}
+
+checkRequiredPrograms() {
+    for p in "${REQUIRED_PROGRAMS[@]}"
+    do
+        verifyProgram $p
+    done
+
+    if ! $STARTUP_CHECK_SUCCESS
+    then
+        echo Please install missing applications
+        exit
+    fi
+}
+
+checkRequiredPrograms
+
```

Comment:
Please revert this and use the downloaded plantuml. We need all developers to use the same version since the output changes from version to version

Reply:
I changed this since the download approach was broken. I just got the error no such file or directory. But I see what I can do with the fixed version.

Reply:
I just ran the script and it seems to work fine. Maybe a local problem